packages/web-twig/src/Resources/components/Toast/stories/ToastDynamicToastQueue.twig

```diff
@@ -13,9 +13,18 @@
         toastElement.classList.remove('Toast--top', 'Toast--bottom');
         toastElement.classList.add(`Toast--${event.target.value}`);
     };
+
+    const setToastCollapsing = (event, selector) => {
+        const toastElement = document.querySelector(selector);
+
+        toastElement.classList.toggle('Toast--collapsible');
+    };
 </script>
 
 <form>
+    <fieldset onchange="setToastCollapsing(event, '#toast-example')" style="border: 0;">
+        <Checkbox name="is-collapsible" autocomplete="off" id="toast-is-collapsible" isChecked label="Collapsible" />
```

Comment:
There is something wrong with `Checkbox` since I fixed it 3 weeks ago in #1343…

<img width="447" alt="obrazek" src="https://github.com/lmc-eu/spirit-design-system/assets/5614085/af66e193-61d3-419e-ac92-403e1ed54883">

(Mark the duplicate double quotes at the end)

Reply:
Please fix it elsewhere.

Reply:
Fixed by #1383.